Minor cleanup and documentation

IzzySoft opened the pull request:
Here's my initial PR (as we've talked about). I've kept commits "atomar" again, so you can easily cherry-pick if you don't like to take it in full :)

IzzySoft committed:
include name of module with "done notification"

IzzySoft committed:
coding style: don't mix tabs with spaces for indention

IzzySoft committed:
updated documentation

SebKuzminsky commented on bin/backup:
```diff
@@ -172,16 +172,16 @@ while getopts 'hc:' ARG; do
     case ${ARG} in
         'h')
             cat <<-"EOF"
-			usage: backup [OPTIONS] [MODULE...]
```
The tabs are actually syntactic here: https://www.gnu.org/software/bash/manual/bashref.html#Here-Documents

After this change, the backup script no longer runs:

> ./bin/backup: line 232: warning: here-document at line 174 delimited by end-of-file (wanted `EOF')
> ./bin/backup: line 233: syntax error: unexpected end of file

I'm using bash 4.3.30, fwiw.

SebKuzminsky commented:
The docs are much better, but the -h change breaks things.  If you rebase your PR and remove that commit, I'll merge it.

IzzySoft committed:
revert changes from 9c4a42e (broke script)

IzzySoft committed:
include name of module with "running backup notification" (analog 9c4a42e)

IzzySoft commented:
Sorry for that, @SebKuzminsky – I should have tested it though it seemed like a "trivial cleanup". I've reverted that commit, and also updated the missing module announcement when starting backup. Not sure what you've meant by "-h change" – but I guess it's that. Let me know if it's OK now: this time I've tested it, and it works as expected.

SebKuzminsky commented:
The new docs look great, thanks!

IzzySoft commented:
Glad you like it! And I've just learned about `<<-` and tabs in HERE documents – something I didn't knew before but comes in real handy! Though I still don't understand how replacing the tabs by blanks broke it (that might require a closer look which I currently lack the time for).

Thanks for merging!
